extensions/amp-inline-gallery/1.0/README.md

```diff
@@ -17,7 +17,17 @@ import {defineElement as defineBentoInlineGallery} from '@bentoproject/inline-ga
 defineBentoInlineGallery();
 ```
 
-### Example: Include via `<script>`
+### Include via `<script>`
+
+```html
+<script type="module" src="https://cdn.ampproject.org/bento.mjs"></script>
+<script nomodule src="https://cdn.ampproject.org/bento.js"></script>
+<script type="module" src="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.mjs"></script>
+<script nomodule src="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.js"></script>
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.css">
```

Comment:
Speaking of which, it seems `type="text/css"` [isn't needed either](https://stackoverflow.com/a/5409146/93579), as it is the default in HTML5 (and browsers didn't require it before either).

```suggestion
<link rel="stylesheet" href="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.css">
```